projects/fsastorefrontlib/src/core/my-account/facade/claim.service.ts

```diff
@@ -115,18 +117,16 @@ export class ClaimService {
   }
 
   createClaim(policyId: string, contractId: string) {
-    this.userIdService
-      .getUserId()
-      .pipe(take(1))
-      .subscribe(occUserId =>
-        this.store.dispatch(
-          new fromAction.CreateClaim({
-            userId: occUserId,
-            policyId: policyId,
-            contractId: contractId,
-          })
-        )
+    this.oboCustomerService
```

Comment:
Again, if possible refactor this so it's not subscribing and unsubscribing here in the service.